draft-ietf-keytrans-protocol.md

```diff
@@ -288,74 +288,96 @@ correctly.
 
 Each leaf node in a prefix tree represents a specific mapping from search key to
 value, while each parent node represents some prefix which all search keys in
-the subtree headed by that node have in common. The subtree headed by a parent's
-left child contains all search keys that share its prefix followed by an
-additional 0 bit, while the subtree headed by a parent's right child contains
-all search keys that share its prefix followed by an additional 1 bit.
-
-The root node, in particular, represents the empty string as a prefix. The
-root's left child contains all search keys that begin with a 0 bit, while the right
-child contains all search keys that begin with a 1 bit.
+the subtree headed by that node have in common. The prefix that a parent node
+represents is based on the its **midpoint** and the **first unspecified
+byte** of the search key.
+
+The first unspecified byte of a search key is the first byte that has not been
+uniquely identified in the course of searching the tree. Initially, this will be
+the leftmost byte of the search key. The midpoint of a parent node is a byte
+value, chosen by the Transparency Log, that bisects the possible values for the
```

Comment:
```suggestion
value chosen by the Transparency Log such that it bisects the possible values for the
```

"Chosen by the transparency log" sounds like the log can choose arbitrary values. Is this phrasing more accurate?

Reply:
It can in fact choose arbitrary values. So an evil Transparency Log can make search paths longer